style.css

```diff
@@ -0,0 +1,203 @@
+* {
+  padding: 0;
+  margin: 0;
+}
+
+html {
+  box-sizing: border-box;
+}
+
+* *::before *::after {
+  box-sizing: inherit;
+}
+
+body {
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  margin: 10px;
+  padding: 10px;
+  border: 20px;
+  background-image: url(images/Header.svg);
+  background-repeat: no-repeat;
+  max-height: 100vh;
+  max-width: auto;
+}
```

Comment:
[OPTIONAL] Your font-size seems to be smaller compared to the actual design, kindly ensure that you use all the stated dimensions in the design to help you achieve a more responsive and appealing site. :+1: